src/olympia/blocklist/models.py

```diff
@@ -220,14 +224,20 @@ def is_save_to_blocks_permitted(self):
         )
 
     def save(self, *args, **kwargs):
-        if self.input_guids and not self.processed_guids:
+        def serialize_block(block):
+            return {
+                'id': block.id,
+                'guid': block.guid,
+                'adu': block.addon.average_daily_users,
```

Comment:
Can we rename `adu` to `average_daily_users` everywhere? It is super confusing, especially given that Mozilla uses acronyms like DAU and ADAU (for daily active users).

Reply:
https://github.com/mozilla/addons-server/pull/13300/commits/c5d5002fe1ae86db2c0dbbf84bf2d2c0f919b75d